### DISCUSSION TITLE
Support for a dedicated notifications queue

### DISCUSSION THREAD

**Discussion by kofiasare** [excid3/noticed#324] [General]
@excid3 Thank you for this gem.

I use `sidekiq` as my active_job queue_adapter. 

Is there a way I can tell noticed to use a queue other than the `default`  queue ?.

**Comment by kofiasare** [comment]
Figured it out.  It appears this not documented. 

pass `queue: :queue_name` to delivery method options

i.e: `  deliver_by :stdout, class: 'DeliveryMethods::Stdout',  queue: :notifications`

But still it will be nice to make all notifications run on a dedicated queue

**Comment by bobmaerten** [comment]
@excid3, would it be fine to add an initializer like this?

```ruby
# config/initializers/noticed.rb

Noticed::DeliveryMethods::Base.queue_name = "not_default_queue_name"
```

**Comment by excid3** [comment]
Should be fine, but you may want to do that on to_prepare so that if code reloads in development it runs it again.
